data/resources/ui/login-code.blp

```diff
@@ -0,0 +1,69 @@
+using Gtk 4.0;
+using Adw 1;
+
+template LoginCode {
+  layout-manager: BoxLayout {
+    orientation: vertical;
+  };
+  vexpand: true;
+
+  Adw.HeaderBar {
+    styles ["flat"]
```

Comment:
Flat headerbars are planned to be deprecated. You should now use an [AdwToolbarView](https://gnome.pages.gitlab.gnome.org/libadwaita/doc/main/class.ToolbarView.html) to have that new flat style.

Reply:
Yes, I had this for some time, but did not push it.